test/AppLifecycle/FunctionalTests.cpp

```diff
@@ -35,33 +46,37 @@ namespace ProjectReunionCppTest
         const std::wstring c_testPackageFullName = L"AppLifecycleTestPackage_1.0.0.0_x64__ph1m9x8skttmg";
 
     public:
-        BEGIN_TEST_CLASS(AppLifecycleFunctionalTests)
+        BEGIN_TEST_CLASS(AppLifecycleTests)
+            TEST_CLASS_PROPERTY(L"IsolationLevel", L"Method")
             TEST_CLASS_PROPERTY(L"ThreadingModel", L"MTA")
-            TEST_CLASS_PROPERTY(L"RunFixtureAs:Class", L"InteractiveUser")
+            TEST_CLASS_PROPERTY(L"RunAs:Class", L"RestrictedUser")
         END_TEST_CLASS()
 
         TEST_CLASS_SETUP(ClassInit)
         {
-            // Deploy packaged app to register handler through the manifest.
-            //RunCertUtil(c_testPackageCertFile);
-            //InstallPackage(c_testPackageFile);
+            WEX::Logging::Log::Comment(L"CLASS_SETUP---Begin---");
```

Comment:
Do we really need these since TAEF spits out that it's running them already?

I find this command useful after making tons of changes to cut through the mess. :)

git difftool -d 37724ef1072e35943487d165dd2e66d33ccb3681..

Reply:
I normally don't put in such blazing markers but trobuleshooting bit test output (when they ran) from CI/Pipeline was eyeblurring. Past that point so will remove the printf aids.